Curs04/src/views/index.js

```diff
@@ -1,8 +1,96 @@
 
 window.onload = function(){
-    const result = [];
-    result.push(LoginPage({}))
     const app = document.querySelector('#app');
+    // call LoginPage function and display LoginPage using innerHTML method
     app.innerHTML = LoginPage();
+
+    // get the form by his ID
+   const loginFormSection = document.getElementById("form-login")
+
+    const myObj = {}
+
+   const goToNewEvaluationPage = function(event) {
+        event.preventDefault();
+        // call NewEvaluationPage() which will render the New-evaluation page
+        app.innerHTML = NewEvaluationPage()
+
+        // get evaluationButton by his ID
+        const evaluationButton = document.getElementById("evaluation-page")
+
+        // add "click" event listener
+        evaluationButton.addEventListener("click", goToEvaluationsPage)
+
+        const getSelectorAll = document.getElementsByTagName("select")
+        getSelectorAll.forEach(function(el) {
+           const nameSelector = el.name
```

Comment:
do we really need these constants?

Reply:
I use this constants to select all the selects and to select their name and value.

Probably exists other ways...